index.bs

```diff
@@ -249,20 +252,36 @@ and defensive programming which includes:
     enhanced by the possible usage of a sensor, not degraded by its
     absence.
 
-<div class="example">
+<div class="example" id="robust-example">
+    The following snippet illustrates how an Accelerometer sensor can be created
+    in a robust manner.
     <pre highlight="js">
-        try { // No need to feature detect thanks to try..catch block.
-            var sensor = new GeolocationSensor();
-            sensor.start();
-            sensor.onerror = error => gracefullyDegrade(error);
-            sensor.onreading = _ => updatePosition(sensor.latitude, sensor.longitude);
-        } catch(error) {
-            gracefullyDegrade(error);
+        let accelerometer = null;
+        try {
+            accelerometer = new Accelerometer({ frequency: 10 });
+            accelerometer.addEventListener('error', event => handleErrors(event.error));
+            accelerometer.addEventListener('reading', () => reloadOnShake(accelerometer));
+            accelerometer.start();
+        } catch (error) {
+            handleErrors(error);
+        }
+
+        function handleErrors(error) {
```

Comment:
Its called handleErrors but not doing much handling :-) more like logging

Reply:
Well, it is an example and not a full blown app :) Do you have some suggestions? It is app specific, to define behavior for particular error condition. E.g, fallback to DevMotion, show notification, etc.